testgres/operations/local_ops.py

```diff
@@ -0,0 +1,280 @@
+import getpass
+import os
+import shutil
+import subprocess
+import tempfile
+from shutil import rmtree
+
+import psutil
+
+from testgres.exceptions import ExecUtilException
+from testgres.logger import log
+
+from .os_ops import OsOperations
+from .os_ops import pglib
+
+try:
+    from shutil import which as find_executable
+except ImportError:
+    from distutils.spawn import find_executable
+
+CMD_TIMEOUT_SEC = 60
+
+
+class LocalOperations(OsOperations):
+    def __init__(self, host='127.0.0.1', hostname='localhost', port=None, username=None):
```

Comment:
seems it's better to extract ```host, hostname, port, username``` to params object, according to my commentary to ```ssh_key``` above